src/packages/auth/src/authentication/apollo/plugin.ts

```diff
@@ -58,6 +63,7 @@ export const authApolloPlugin = (
 
 			// We may need to return a redirect to the client. If so, we'll set this variable.
 			const authHeader = request.http?.headers.get('authorization');
+			const apiKeyHeader = request.http?.headers.get('X-API-Key');
```

Comment:
Is there anything special about 'X-API-Key'? What is X?

Reply:
`x` is used to indicate that this is not a standard HTTP header.